README.md

```diff
@@ -365,6 +366,20 @@ Usage: `akamai list-contracts`
 | - | - |
 | -h, --help  | output usage information |
 
+### List Properties
+List of properties associated to a specific EdgeWorker Id.
+
+Usage: `akamai edgeworkers list-properties <edgeworkerId> [options]`
+
+| Option | Description |
+| - | - |
+| -h, --help  | output usage information |
+| --activeOnly | Returns only active properties |
+
+| Argument | Existence | Description |
+| - | - | - |
+| edgeworkerId | required | Edgeworker identifier.
```

Comment:
"edgeworkerId" and the one listed above "lp `<edgeworker-identifier> [options]`" are different. can we keep it same ?